installer/windows/agent-installer.nsi

```diff
@@ -58,11 +58,16 @@ function .onInit
 	${GetParameters} $R0
     ${GetOptions} $R0 ~OPENBAS_URL= $ConfigURL
     ${GetOptions} $R0 ~ACCESS_TOKEN= $ConfigToken
+    ${GetOptions} $R0 ~NON_SYSTEM_USER= $ConfigNonSystemUser
+    ${GetOptions} $R0 ~NON_SYSTEM_PWD= $ConfigNonSystemPwd
 functionEnd
 
```

Comment:
Finally, do we let the installer create the user account ?
Not the agent to have too much power

This facilitates the agent deployment workflow.

Reply:
No, these params are not used to create a user account, they are used to execute scripts (dep. Elevated privilege)